packages/server/src/controllers/UsersController.js

```diff
@@ -1,172 +1,157 @@
-import Models from '../models'
 import HttpError from '../errors/http-error'
 import UsersValidator from '../validators/UsersValidator'
-
-const { User, UserAddress } = Models
-
-// temp dummy models
-let Users = { }
-let UserEmails = { }
-let UserSessions = { }
+import Users from '../models/objection/Users'
+import UserEmails from '../models/objection/UserEmails'
 
 
 export default {
   async details(req, res) {
     const { params: { address } } = req
-    const body = {
-      emailExists: false,
-      emailVerified: false,
-      addressVerified: false,
-      notificationsDisabled: false,
-      ... Users[address]
-    }
-    res.send(body)
+    const user = await Users.query().findOne({address}).withGraphFetched('[email, emailVerificationToken, notificationSettings]')
```

Comment:
Not sure about this since those relations are doing multiple joins. I prefer to be more explicit about what returns I am doing.

I think other abstraction suggestions are good, we can have a rule for creating abstract queries that include logic, I added a few.